src/debian/ssh-audit_hardening.conf.j2

```diff
@@ -32,4 +32,5 @@ Compression no
 ClientAliveInterval 300
 ClientAliveCountMax 0
 PermitTunnel no
-PermitOpen none
\ No newline at end of file
+PermitOpen none
+ListenAddress {{ ansible_env.SSH_CONNECTION.split(' ')[2] }}
```

Comment:
You cannot use this address. We do not know which address the SEAPATH user wants to bind sshd.
For me, this information should be described in the Ansible inventory.

Reply:
Commit moved out of this PR.